fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -1091,21 +1126,37 @@ public Builder setIncreaseLimitAfter(int increaseLimitAfter) {
              */
             @Nonnull
             public Builder setTimeLimitMilliseconds(long timeLimitMilliseconds) {
-                if (timeLimitMilliseconds < UNLIMITED_TIME) {
+                if (timeLimitMilliseconds < 0) {
                     timeLimitMilliseconds = UNLIMITED_TIME;
                 }
                 this.timeLimitMilliseconds = timeLimitMilliseconds;
                 return this;
             }
 
+            /**
+             * Set the time quota for a single transaction. If this quota is exceeds, the indexer will commit the
+             * transaction and start a new one. This can be useful to avoid timeouts while scanning a high number of records
+             * in each transaction.
```

Comment:
You mention "timeouts" here, but the more relevant thing to mention might be the FDB five second transaction limit: https://apple.github.io/foundationdb/known-limitations.html#long-running-transactions

Mentioning that limitation of FDB might provide some guidance for people on how to set this value

Reply:
✅